tests/ci_build/Dockerfile.ubuntu1404_crosstool

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
+MAINTAINER Aran Khanna <arankhan@amazon.com>
+
+# UPDATE BOX
+RUN apt-get update && apt-get -y upgrade
+
+# TOOLCHAIN DEPS
+RUN apt-get install -y python python-setuptools python-pip python-dev unzip gfortran
+RUN apt-get install -y git bison cvs flex gperf texinfo automake libtool help2man make libtool-bin libncurses5-dev g++ cmake wget gawk
+RUN pip install numpy nose
+
+# BUILD TOOLCHAIN
+RUN git clone https://github.com/arank/crosstool-NG
+RUN cd crosstool-NG && ./bootstrap && ./configure && make && make install
+
+RUN useradd -ms /bin/bash aran
+RUN cd && cp -R .profile .bashrc /home/aran
+ADD . /home/aran/build
+
+RUN chown -R aran:aran /home/aran
+
+RUN cd /home/aran/build; su -m aran -c "ct-ng arm-unknown-linux-gnueabi && ct-ng build"
```

Comment:
don't add a new user, if you look at `https://github.com/dmlc/mxnet/blob/master/tests/ci_build/with_the_same_user`, it will add and run with user who started the docker container